simulators/ethereum/sync/sync.go

```diff
@@ -43,17 +43,46 @@ func main() {
 		Description: `This suite of tests verifies that clients can sync from each other in different modes.
 For each client, we test if it can serve as a sync source for all other clients (including itself).`,
 	}
+
+	// SYNC by ETH protocol
+	ethParams := params.Set("HIVE_NODETYPE", "full")
 	suite.Add(hivesim.ClientTestSpec{
 		Name:        "CLIENT as sync source",
 		Description: "This loads the test chain into the client and verifies whether it was imported correctly.",
-		Parameters:  params,
+		Parameters:  ethParams,
 		Files:       sourceFiles,
-		Run:         runSourceTest,
+		Run:         runETHSyncTest,
 	})
-	hivesim.MustRunSuite(hivesim.New(), suite)
+
+	// SYNC by LES protocol
+	//
+	// Add the light client sync test. This only works with geth for now.
+	var (
+		clients []string
+		sim     = hivesim.New()
+	)
+	clientTypes, err := sim.ClientTypes()
+	if err != nil {
+		panic(err)
+	}
+	for _, name := range clientTypes {
+		if !strings.HasPrefix(name, "go-ethereum") {
+			continue
+		}
```

Comment:
Would make sense to filter out the geth client less than 1.9.25

Reply:
We can't really do that here because the client version is not available to the simulator.